rpc/ut/link_ut.c

```diff
@@ -402,6 +404,54 @@ void rlut_connect_async()
 	stop_rpc_client_and_server();
 }
 
+/*
+ * NOTE: already tried to generalise rlut_connect_async() with no luck therefore
+ * rlut_connect_async_op() was introduced.
```

Comment:
Please, add also a few words what's the difference between rlut_connect_async() and rlut_connect_async_op().